axelrod/strategies/lookerup.py

```diff
@@ -232,10 +233,12 @@ class LookerUp(Player):
 
     The dictionary must contain all possible permutations of C's and D's.
 
-    LookerUp can also be instantiated with pattern=str/tuple of actions, and
-    parameters=Plays(self_plays=player_depth: int,
-                     op_plays=op_depth: int,
-                     op_openings=op_openings_depth: int).
+    | LookerUp can also be instantiated with pattern=str/tuple of actions, and
```

Comment:
what is a pipe? you mean like in bash?

the docstrings use rst notation, correct?  I thought it would be useful to preserve the line-breaks there.  so i used that.  without the "hey rst! preserve linebreaks"-notation, it threw an error with one of the auto checks for having naughty indentation.  should i change to "::" notation?  

while i think of it,  can i go ahead and delete all the commented-out original data?